### DISCUSSION TITLE
Latest beta version & changes

### DISCUSSION THREAD

**Discussion by ineuw** [autokey/autokey#644] [Beta feedback]
I am using beta-9. I've noticed some changes in the editor behaviour. which is the current beta?

**Comment by josephj11** [comment]
When you want to answer this, go [here](https://github.com/autokey/autokey/releases/) and see what's there.
The latest is beta-10. I remember that initially, there were some problems with this beta, but it's been quiet recently, so it may be OK now.

**Reply by ineuw** [reply]
Many thanks. Now, I have a chance test some recent anomalies with Autokey and track down the source. If the issues were not resolved, I will post it in Gitter.
